cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,73 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import ArticlePageObject from '../support/pages/article.pageObject';
+import EditorPageObject from '../support/pages/editor.pageObject';
 
-  });
+const homePage = new HomePageObject();
+const profilePage = new ProfilePageObject();
+const articlePage = new ArticlePageObject();
+const editorPage = new EditorPageObject();
+
+describe('Article', () => {
+  let user;
+  let article;
+  let newTitle;
 
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.register(user.email, user.username, user.password);
+      cy.login(user);
+    });
+
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+      newTitle = article.title + ' new';
+    });
+
+    cy.visit('/');
   });
 
   it('should be created using New Article form', () => {
-
+    homePage.clickOnNewArticleLink();
+    editorPage.typeArticleTitle(article.title);
+    editorPage.typeArticleDescription(article.description);
+    editorPage.typeArticleBody(article.body);
+    editorPage.typeArticleTag(article.tag);
+    editorPage.clickOnPublishArticleButton();
+    articlePage.assertArticleIsPublished(article.title);
   });
 
   it('should be edited using Edit button', () => {
-
+    cy.createArticle(article.title, article.description, article.body)
+      .then((response) => {
+        const slugUr = response.body.article.slug;
```

Comment:
There is a typo in the variable name `slugUr`. It should likely be `slugUrl` to better reflect its purpose as a URL slug.